tests/test_load_from_msp.py

```diff
@@ -0,0 +1,14 @@
+import os
+from matchms.importing import load_from_msp
+
+
+def test_load_from_msp():
+
+    module_root = os.path.join(os.path.dirname(__file__), "..")
+    spectrums_file = os.path.join(module_root, "tests", "MoNA-export-GC-MS.msp")
```

Comment:
It would be better to use a smaller file as example (and for testing). Can you make a small subset of the all-MoNA export? Say, a "MoNA-GC-MS-first10.msp" which only contains the first 10 spectra? (or something similar)

Reply:
Sure, I will create another file.